ci/update-jetpack.js

```diff
@@ -25,6 +26,54 @@ function incrementVersion(version) {
     return result;
 }
 
+function parseVersion( version ) {
+    return version.split(/[.-]/).map(part => isNaN(Number(part)) ? part : Number(part));
```

Comment:
```suggestion
    return version.split(/[.-]/).map(part => isNaN(+part) ? part : +part);
```

Reply:
I actually think the explicit casting there is more easily readable than the implicit one. Or is there some other reason for preferring the suggested way?